modules/vba_documents/spec/fixtures/valid_metadata.json

```diff
@@ -1,7 +1,7 @@
 { 
   "veteranFirstName": "Jane", 
   "veteranLastName": "Doe", 
-  "fileNumber": "012345678", 
+  "fileNumber": "  012345678 ", 
```

Comment:
Rather than modifying this file which is intended to be the "model" version of metadata, have you considered modifying the value in the spec file prior to using it instead? Something like this:
```
# code that sets up the test, including the metadata (using valid_metadata.json)
metadata['fileNumber'] = "  012345678 "
# code that runs the UploadProcessor and confirms after that the fileNumber was sent as "012345678"
```

Reply:
reverted